src/accounts/templates/accounts/emails/email_change_completed_new_body.html

```
@@ -0,0 +1,17 @@
+{% load i18n %}<!DOCTYPE html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)